ext/openssl/ossl_pkey.c

```diff
@@ -628,6 +628,71 @@ ossl_pkey_initialize_copy(VALUE self, VALUE other)
 }
 #endif
 
+/*
+ *  call-seq:
+ *      OpenSSL::PKey.private_new(algo, string) -> PKey
+ *
+ * See the OpenSSL documentation for EVP_PKEY_new_raw_private_key()
+ */
+
+#ifdef HAVE_EVP_PKEY_NEW_RAW_PRIVATE_KEY
```

Comment:
I think `#ifdef` needs to be moved above the doc comment so that rdoc can it correctly.

Reply:
Fixed @ 202cac7 .
Maybe the first and second `#ifdef` blocks can be merged into one, but I thought having it separated would make it easier to understand that each method needs this condition to be met.